### ISSUE TITLE
Problems using whiskers with real papers

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/paper-land#99]
@brettfiedler found that whiskers are not working with real papers. After some debugging we found two problems

1) The function updating whisker collisions pulls from `paperProgramInfo.number` (which is a string) and uses that value in the various program data maps whose keys need to be of type number (a typing issue).

2) The intersection events didn't work as expected when the position update interval was high. Consider making the intersection updates happen more frequently than the position intervals.

**Comment by jessegreenberg** [comment]
This should be fixed soon as we would like it to work well for demonstration purposes.

**Comment by jessegreenberg** [comment]
OK, I think the above commit fixes this. I was able to test by putting a monitor with images of paper in front of my web cam, and I was able to see whisker-demo-1 working. It was a little buggy, but I *think* that is because of the bad detection and not the whisker parts of the code.

> The intersection events didn't work as expected when the position update interval was high. Consider making the intersection updates happen more frequently than the position intervals.

Coming back to this, I am wondering if this is expected? It made it harder to find this bug, but it seems whiskers should be resilient to jitter too? Maybe there is a problem when position interval is larger than whisker length. But maybe an easy fix for this is to not let the position interval get larger than 0.05, or put a warning in the UI. Just brainstorming here, I am not sure what is best.
